external/CMakeLists.txt

```diff
@@ -54,6 +54,9 @@ set(OLP_SDK_CPP_RAPIDJSON_TAG "master")
 set(OLP_SDK_CPP_BOOST_URL "https://github.com/boostorg/boost.git")
 set(OLP_SDK_CPP_BOOST_TAG "boost-1.72.0")
 
+set(OLP_SDK_CPP_LMDB_URL "https://github.com/LMDB/lmdb.git")
```

Comment:
Forgot to mention, please update above the year to `2019-2021`

Reply:
OK